DirectEmployers/MyJobs
# Remove reference to nonexistent template

## What's this PR do?
Removes a reference to a template that was removed some time ago.
## Where should the reviewer start?
It's one line.
## How should this be manually tested?
Hit the activation page on your dev server. If debug is enabled, you should not see a yellow screen. If debug is disabled, you should not see an error in your runserver log.
## Any background context you want to provide?
We removed a template. We did not remove references to it.
## PR Checklist

### Front-End
